gui/src/test/java/kcl/teamIndexZero/traffic/gui/GUI_PrimitivesTest.java

```diff
@@ -0,0 +1,48 @@
+package kcl.teamIndexZero.traffic.gui;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by kumarawijeet on 3/16/2016.
+ */
+public class GUI_PrimitivesTest {
```

Comment:
Already tried that manually, by giving false name for the file. Will get pics to show if needed, it basically shows an error.                                                                                                                                                                                                                                                                                                                                        Sent from my BlackBerry 10 smartphone on the EE network.                                                                                                                                                                                                                From: EsSent: Monday 21 March 2016 00:57To: teamIndexZero/index_zero_trafficsystemReply To: teamIndexZero/index_zero_trafficsystemCc: kumar awijeetSubject: Re: [index_zero_trafficsystem] Gui graphics kumar awijeet (#9)In gui/src/test/java/kcl/teamIndexZero/traffic/gui/GUI_PrimitivesTest.java:
> @@ -0,0 +1,48 @@
> +package kcl.teamIndexZero.traffic.gui;
> +
> +import org.junit.After;
> +import org.junit.Before;
> +import org.junit.Test;
> +
> +import static org.junit.Assert.*;
> +
> +/**
> + * Created by kumarawijeet on 3/16/2016.
> + */
> +public class GUI_PrimitivesTest {

Testing that your exceptions are thrown in the right circumstances (i.e. when the file is either corrupted or absent) would be a good thing to do here I would think.

—You are receiving this because you authored the thread.Reply to this email directly or view it on GitHub